sample/AspNetFullFrameworkSampleApp/AspNetFullFrameworkSampleApp.csproj

```diff
@@ -4,6 +4,7 @@
   </Target>
   <Target Name="Clean" Condition="'$(OS)' != 'WINDOWS_NT'">
   </Target>
+  <Import Project="..\..\packages\MSBuild.Microsoft.VisualStudio.Web.targets.14.0.0.3\build\MSBuild.Microsoft.VisualStudio.Web.targets.props" Condition="Exists('..\..\packages\MSBuild.Microsoft.VisualStudio.Web.targets.14.0.0.3\build\MSBuild.Microsoft.VisualStudio.Web.targets.props')" />
```

Comment:
It was deleted in #609, however, it cannot be built locally without such import

Reply:
Oh, you probably don't have ASP.NET installed locally, right?

Reply:
If we talk about Windows features, I have `ASP`, `ASP.NET 3.5` and `ASP.NET 4.8` components

Reply:
I meant the ASP.NET workload - or to be precise "Web Development build tools" when you install Visual Studio. Like mentioned [here](https://stackoverflow.com/a/44064473)

I think the Windows feature only makes you able to host those project, I'm not sure it's the full SDK - but I'm not sure. 

I just would like to understand why this happened to you - it seems working for me locally and also in our CI (which I know has the web dev. build tools). 

Overall I'm ok with merging it - if that helps people without that workload to build then it's definitely better to have it.

Reply:
That folder is not populated by anything in the solution, so if it exists locally for @vhatsura, then it must be because he doesn't have a clean workspace, and has this folder remaining from a previous version of the code. Dead paths should not be added into the project file. This project file was previously updated to use `PackageReference` flavour NuGet, but the work wasn't complete. #609 addresses this by pulling in all required dependencies properly via NuGet. As @gregkalapos mentioned, this works on the CI server.

@vhatsura I would suggest doing a complete clean and rebuild to begin with, and then addressing any missing workloads or SDKs on your system before proceeding.

Reply:
@NeoXtreem, I deleted `packages` folder manually (clean target didn't help) and remove this line. After that I got the following error:
```
  The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Microsoft\VisualStudio\v16.0\WebApplications\Microsoft.WebApplication.targets" was not found. Also, tried to find "WebApplications\Microsoft.WebApplication.targets" in the fallback search path(s) for $(VSToolsPath) - "C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v16.0" . These search paths are defined in "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe.Config". Confirm that the path in the <Import> declaration is correct, and that the file exists on disk in one of the search paths.  C:\projects\apm-agent-dotnet\sample\AspNetFullFrameworkSampleApp\AspNetFullFrameworkSampleApp.csproj
```
Beside I found issue in developer community which is the same as for me: https://developercommunity.visualstudio.com/content/problem/414282/vs-2019-webapplication-projects-fail-to-load.html. So, I think it isn't related to missed some installation packages. BTW, "Web Development build tools" is installed in my setup (I use Rider, however VS 2019 is installed). Downgrade MSBuild version (Rider allow to do it for all solution) to 15.0 helps me to build the project, however, it will be an issue with moving to .Net Core 3.0 support.

Reply:
Thanks for the update! 
I'd say let's ignore it for now, if we ran into this in https://github.com/elastic/apm-agent-dotnet/issues/546 we can still make a change - usually we should prefer these build and project file changes outside of agent code changes anyway.

Reply:
Never mind, it seems I fixed it (I just open VS installer to check installed components and close it). After that, all is working fine. Sorry for inconvenience.